### ISSUE TITLE
Removing password protection

### ISSUE THREAD

**Issue by Bjwebb** [NRGI/resourceprojects.org#34]
Is this something we wa nt to do for an alpha site?

Creating this issue to consider the steps that will need to be involved:
- [ ] Creating a stronger admin password for the virtuoso so that people can't log in as admin

**Comment by timgdavies** [comment]
Could we expose the main site without exposing the Ontowiki and Virtuoso? 

If we wanted to expose a SPARQL endpoint, it might make sense to do that through a reverse proxy on apache, rather than directly exposing all of the Virtuoso docker also.

**Comment by Bjwebb** [comment]
Yes, this is also an option. I suspect as we move to "alpha" stage that we might want a non-default admin password for virtuoso also (since the http username/password are shared pretty widely). If we do that, I guess the question is whether exposing virutoso is easier/better than setting up a reverse proxy.

**Comment by timgdavies** [comment]
Agreed - this is an 'in addition to' rather than an 'instead of'

**Comment by Bjwebb** [comment]
I've just set a stronger password for the virtuoso dba user. The credentials can be found in the company KeePass.

**Comment by Bjwebb** [comment]
I've also set up as a reverse proxy to virtuoso as you suggest, which will be at http://alpha.resourceprojects.org/sparql when the DNS goes live.

Remaining task is just to remove the password protection on alpha.resourceprojects.org (which is lodspeakr plus /sparql from virtuoso), when we're ready to do that.
